www/about/policies/terms-of-service.md

```diff
@@ -38,7 +38,7 @@ these Terms.
 1. General Terms
 
   1. The Service is a platform to facilitate payments for goods or services
-     ("Work").
+     ("Work"). The scope of Work is defined by Projects.
```

Comment:
This gives me an opportunity to fix something that has been bugging me - unclear from this sentence in isolation if the Work is the goods or the payment for the goods. So I might say (also slightly tweaking the new clause):

> The Service is a platform to facilitate payments for goods or services. The nature and scope of such goods or services (the "Work") is defined by each Project.